guides/common/modules/proc_upgrading-smartproxy-server.adoc

```diff
@@ -137,5 +137,44 @@ include::snip_steps-needs-reboot.adoc[]
 
 . Optional: If you made manual edits to DNS or DHCP configuration files, check and restore any changes required to the DNS and DHCP configuration files using the backups made earlier.
 ifdef::orcharhino,satellite[]
-. Optional: If you use custom repositories, ensure that you enable these custom repositories after the upgrade completes.
+. Optional: If you use custom repositories,ensure that you enable these custom repositories after the upgrade completes.
 endif::[]
+
+.Upgrading {SmartProxyServersTitle} Using Remote Execution
```

Comment:
ISG says we need to use sentence case for headers, including "dot" headers. I have started updating the file headers but have not gotten to dot headers yet. Please change to "Upgrading {SmartProxyServersTitle} using remote execution". 

Just as a point of information, if you do use title case for headers, "using" is not capitalized because it is a preposition.